app/assets/stylesheets/donor.css.scss

```diff
@@ -0,0 +1,230 @@
+@mixin clear-list-margins() {
+  margin: 0;
+  list-style-type: none;
+}
+
+@mixin card-header-arrow() {
+  display: inline-block;
+
+  .fa {
+    padding-top: 12px;
+  }
+
+  a {
+    color: $dark-grey;
+    &:hover {
+      color: $white;
+    }
+  }
+}
+
+
+.profile-summary-donate-link {
+  background: $white;
+  border: 1px solid $donor-profile-border-grey;
+  border-radius: 4px;
+  color: $medium-green;
+  margin: 0 auto;
+  padding: 5px 10px;
+}
+
+.update-profile-link {
+  background: $donor-profile-light-grey;
+  color: $donor-profile-darker-grey;
+  padding: 8px;
+  text-decoration: none;
+  width: 100px;
+}
+
+.donor-dashboard-block {
+  @include full-width-block($off-white);
+  @include block-padding();
+  min-height: 900px;
+}
+
+.dashboard {
+  border: 1px solid $dark-grey;
+  @include drop-shadow(1);
+}
+
+.card-header {
+  @extend .text-center;
+  border-bottom: 1px solid $dark-grey;
+  background-color: $medium-green;
+
+  .card-header-title {
+    display: inline-block;
```

Comment:
Properties should be ordered color, display, font-size, font-weight, margin-bottom, margin-top